### PR TITLE
Add FingerprintJS Pro API

### PR DESCRIPTION
**Pull request by makma**
- [X] My submission is formatted according to the guidelines in the [contributing guide](/CONTRIBUTING.md)
- [X] My addition is ordered alphabetically
- [X] My submission has a useful description
- [X] The description does not end with punctuation
- [X] Each table column is padded with one space on either side
- [X] I have searched the repository for any relevant issues or pull requests
- [X] Any category I am creating has the minimum requirement of 3 items
- [X] All changes have been [squashed][squash-link] into a single commit

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -911,6 +911,7 @@ API | Description | Auth | HTTPS | CORS |
 | [CRXcavator](https://crxcavator.io/apidocs) | Chrome extension risk scoring | `apiKey` | Yes | Unknown |
 | [Escape](https://github.com/polarspetroll/EscapeAPI) | An API for escaping different kind of queries | No | Yes | No |
 | [FilterLists](https://filterlists.com) | Lists of filters for adblockers and firewalls | No | Yes | Unknown |
+| [FingerprintJS Pro ](https://dev.fingerprintjs.com/docs) | Fraud detection API offering highly accurate browser fingerprinting | `apiKey` | Yes | Yes |
```
Comment (matheusfelipeog) [inline README.md:914]: Remove this space in the name

```suggestion
| [FingerprintJS Pro](https://dev.fingerprintjs.com/docs) | Fraud detection API offering highly accurate browser fingerprinting | `apiKey` | Yes | Yes |
```

**Review by matheusfelipeog — CHANGES_REQUESTED** [review]
Thanks for your contribution @makma :)

Can you resolve this review, please?

**Comment by matheusfelipeog** [comment]
When finished, [squash](https://github.com/todotxt/todo.txt-android/wiki/Squash-All-Commits-Related-to-a-Single-Issue-into-a-Single-Commit) all changes into a single commit, please 🤝

**Comment by makma** [comment]
Hi @matheusfelipeog , sorry for a silly typo, it's already fixed and squashed. Just out of curiosity - do you think I can create a new category in the list for `Fraud Detection`? I believe there are at least 3 APIs that could fit into this category (which should be aligned with the contribution guidelines). Thank you!

**Review by matheusfelipeog — APPROVED** [review]
> Hi @matheusfelipeog , sorry for a silly typo, it's already fixed and squashed.

No problem. Now it's ok, a big THANKS ;)

**Comment by matheusfelipeog** [comment]
> do you think I can create a new category in the list for Fraud Detection? I believe there are at least 3 APIs that could fit into this category (which should be aligned with the contribution guidelines). Thank you!

@makma Yes, you can do this. As long as you have at least 3 APIs for this new section.

Request my review when you do this, please.

**Pull request merged**